.github/workflows/test.yaml

```
@@ -134,6 +134,12 @@ jobs:
           # Just Python 3.12 with default settings.
           - python-version: '3.12'
 
+          # Newest macOS and oldest Python (major) supported versions.
+          - python-version: '3.8'
```

maybe try a very old 3.8

```suggestion
          - python-version: '3.8.0'
```